scala-interpreter/src/main/scala-2.11/org/apache/toree/kernel/interpreter/scala/ScalaInterpreterSpecific.scala

```diff
@@ -258,14 +258,13 @@ trait ScalaInterpreterSpecific extends SettingsProducerLike { this: ScalaInterpr
    * Starts the interpreter, initializing any internal state.
    * @return A reference to the interpreter
    */
-  override def start(): Interpreter = {
+  override def start(): ScalaInterpreter = {
```

Comment:
Was the change in return type intentional? Doesn't this break the override?

Reply:
It doesn't and its kind of handy in my startInit method.